cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -73,6 +75,7 @@ type CMServer struct {
 	DeletingPodsQps         float32
 	DeletingPodsBurst       int
 	ServiceAccountKeyFile   string
+	ClientCAFile            string
```

Comment:
This is the file used by the client to verify the server, which maps to RootCAs in the tls.Config struct:
```
        // RootCAs defines the set of root certificate authorities
        // that clients use when verifying server certificates.
        // If RootCAs is nil, TLS uses the host's root CA set.
        RootCAs *x509.CertPool
```